lib/crypto/src/crypto.erl

```diff
@@ -47,533 +56,572 @@
+]).
+
+-export([
+    stream_init/2, stream_init/3,
+    stream_encrypt/2,
+    stream_decrypt/2,
+    block_encrypt/3, block_encrypt/4,
+    block_decrypt/3, block_decrypt/4
+]).
+
+-deprecated_type([
+    {retired_cbc_cipher_aliases, 0, "Use aes_*_cbc or des_ede3_cbc"},
+    {retired_cfb_cipher_aliases, 0, "Use aes_*_cfb8, aes_*_cfb128 or des_ede3_cfb"},
+    {retired_ctr_cipher_aliases, 0, "Use aes_*_ctr"},
+    {retired_ecb_cipher_aliases, 0, "Use aes_*_ecb"}
+]).
 
 %%%----------------------------------------------------------------
 %% New interface
--export([crypto_init/4, crypto_init/3,
-         crypto_update/2,
-
-         crypto_one_time/4, crypto_one_time/5,
-         crypto_one_time_aead/6, crypto_one_time_aead/7,
-
-         crypto_dyn_iv_init/3,
-         crypto_dyn_iv_update/3,
-         crypto_final/1,
-         crypto_get_data/1,
-
-         supports/1,
-         mac/3, mac/4, macN/4, macN/5,
-         mac_init/2, mac_init/3, mac_update/2, mac_final/1, mac_finalN/2
-        ]).
+-export([
+    crypto_init/4, crypto_init/3,
+    crypto_update/2,
+
+    crypto_one_time/4, crypto_one_time/5,
+    crypto_one_time_aead/6, crypto_one_time_aead/7,
+
+    crypto_dyn_iv_init/3,
+    crypto_dyn_iv_update/3,
+    crypto_final/1,
+    crypto_get_data/1,
+
+    supports/1,
+    mac/3, mac/4,
+    macN/4, macN/5,
+    mac_init/2, mac_init/3,
+    mac_update/2,
+    mac_final/1,
+    mac_finalN/2
+]).
 
 %%%----------------------------------------------------------------
 %% Engine
 -export([
-         engine_get_all_methods/0,
-         engine_load/3,
-         engine_load/4,
-         engine_unload/1,
-         engine_by_id/1,
-         engine_list/0,
-         engine_ctrl_cmd_string/3,
-         engine_ctrl_cmd_string/4,
-         engine_add/1,
-         engine_remove/1,
-         engine_get_id/1,
-         engine_get_name/1,
-         ensure_engine_loaded/2,
-         ensure_engine_loaded/3,
-         ensure_engine_unloaded/1,
-         ensure_engine_unloaded/2
-        ]).
-
--export_type([ %% A minimum exported: only what public_key needs.
-               dh_private/0,
-               dh_public/0,
-               dss_digest_type/0,
-               ec_named_curve/0,
-               ecdsa_digest_type/0,
-               pk_encrypt_decrypt_opts/0,
-               pk_sign_verify_opts/0,
-               rsa_digest_type/0,
-               sha1/0,
-               sha2/0
-             ]).
-
--export_type([engine_ref/0,
-              key_id/0,
-              password/0
-             ]).
+    engine_get_all_methods/0,
+    engine_load/3,
+    engine_load/4,
+    engine_unload/1,
+    engine_by_id/1,
+    engine_list/0,
+    engine_ctrl_cmd_string/3,
+    engine_ctrl_cmd_string/4,
+    engine_add/1,
+    engine_remove/1,
+    engine_get_id/1,
+    engine_get_name/1,
+    ensure_engine_loaded/2,
+    ensure_engine_loaded/3,
+    ensure_engine_unloaded/1,
+    ensure_engine_unloaded/2
+]).
+
+%% A minimum exported: only what public_key needs.
```

Comment:
Oh, hahaha, this one's nice!

Reply:
Added this as a data point to an ongoing discussion here https://github.com/WhatsApp/erlfmt/issues/134